style.css

```diff
@@ -382,3 +412,300 @@ textarea {
+  }
+
+  .presentation {
+    font-size: 20px;
+    margin: 0 auto 12px;
+  }
+
+  .connect {
+    margin: 0 auto 12px;
+    padding: 0;
+  }
+
+  .media-icons {
+    margin: 0 auto 12px;
+  }
+
+  .works {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin: 0 auto;
+  }
+
+  .grid-row {
+    width: 1156px;
+    height: 450px;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
+    grid-template-areas:
+      "card project project"
+      "card specs specs"
+      "card description description"
+      "card tools tools"
+      "card button button"
+      "card blank blank";
+    justify-items: start;
+    border: 1px solid #a7aeff;
+  }
+
+  .grid-row-reversed {
+    width: 1156px;
+    height: 450px;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
+    grid-template-areas:
+      "project project card"
+      "specs specs card"
+      "description description card"
+      "tools tools card"
+      "button button card"
+      "blank blank card";
+    justify-items: start;
+    border: 1px solid #a7aeff;
+  }
+
+  .grid-row:hover {
+    box-shadow: 0 10px 50px 0 #a7aeff;
+    cursor: pointer;
+  }
+
+  .card {
+    grid-area: card;
+    padding: 0 20px;
+  }
+
+  .project {
+    width: 496px;
+    margin-top: 30px;
+    grid-area: project;
+    display: flex;
+    justify-content: start;
+  }
+
+  .project-name {
+    width: auto;
+    font-size: 40px;
+  }
+
+  .project-name,
+  .project-specs,
+  .project-description,
+  .project-tools,
+  .card-button-style {
+    padding-left: 20px;
+  }
+
+  .specs {
+    width: 496px;
+    grid-area: specs;
+    display: flex;
+    justify-content: start;
+  }
+
+  .project-specs {
+    font-size: 18px;
+  }
+
+  .project-description {
+    width: 496px;
+  }
+
+  .description {
+    width: 496px;
+    grid-area: description;
+    display: flex;
+    justify-content: start;
+  }
+
+  .tools {
+    width: 496px;
+    grid-area: tools;
+    display: flex;
+    justify-content: start;
+    font-size: 12px;
+  }
+
+  .card-button {
+    width: 496px;
+    grid-area: button;
+    display: flex;
+    justify-content: start;
+  }
+
+  .blank1 {
+    grid-area: blank;
+  }
+
+  .image-container {
+    width: 544px;
+    height: 450px;
+    display: flex;
+    justify-content: center;
+    margin-top: 0;
+  }
+
+  .image-container img {
+    width: 544px;
+    height: 448px;
+  }
+
+  .about-myself {
+    width: 1156px;
+    height: 460px;
+    margin: 284px auto;
+    display: grid;
+    grid-template-columns: 1fr 2fr;
+    grid-template-rows: 1fr 1fr 1fr 1fr 1fr;
+    justify-items: start;
+    align-items: start;
+    grid-template-areas:
+      "blank languages languages"
+      "title languages languages"
+      "paragraph languages languages"
+      "connect languages languages"
+      "icons languages languages"
+      "button languages languages";
+  }
```

Comment:
- [ ] The about section is great and very similar, but there are some points that makes different from the template. First the background color of the about section should be white, to differentiate from the previous projects section background color and there should be a border radius on the top right corner. Kindly change that.
 - [ ] Also on smaller screen sizes, there is no padding on the sides of this section. Kindly add some padding to match the Template.
 - Your design:
 - 
<img width="969" alt="image" src="https://user-images.githubusercontent.com/61919617/213572523-14b4db0c-df3b-4003-b955-9a345c904a94.png">

 - Figma template:
 - 
<img width="1019" alt="image" src="https://user-images.githubusercontent.com/61919617/213572671-2e948734-9ffb-4e4b-8df7-79e0bbe53bd1.png">